xml/obs_package_formats.xml

```diff
@@ -278,8 +273,45 @@ systemctl enable sshd</screen>
   </example>
  </sect1>
 
- <sect1>
+ <sect1 xml:id="sec.obs.pkgformats.appimage">
   <title>&obs.ai;</title>
   <remark>toms 2017-08-18: TODO What is it and what is needed</remark>
+  <para>
+   An &obs.ai; is a bundled application including libraries that
+   allows to run on a variety of different distributions without
+   further installation or modification. The &obs.ai; contains all
+   libraries that are not already part of the base system.
+   &obs.ai; keeps the underlying operating system untouched, which means,
+   you do not need root permission nor installation.
```

Comment:
permission->permissions
nor->for
(better: `...do not need <systemitem class="user">root</systemitem> permissions to install an &obs.ai;.`)

Reply:
Fixed and used your suggestion in parenthesis.